dev_scripts/find_images_to_rebuild.py

```diff
@@ -40,15 +46,33 @@
 
 errata = Errata()
 kwargs = {}
-if container_images:
+
+if args.container_img:
     EventClass = ManualRebuildWithAdvisoryEvent
-    kwargs['container_images'] = container_images
+    kwargs['container_images'] = args.container_img
 else:
     EventClass = ErrataAdvisoryStateChangedEvent
+
+event_id = None
+if args.cassette:
+    cassette_name = os.path.splitext(os.path.basename(args.cassette))[0]
+    extension = os.path.splitext(os.path.basename(args.cassette))[1]
+    try:
+        event_id = int(cassette_name)
+        if extension is not ".yml":
```

Comment:
`is` will compare the memory address and not if the values are equivalent. For this, use `==`/`!=`.
```suggestion
        if extension != ".yml":
```

This shows the behavior:
```
>>> extension = ".yml"
>>> extension is ".yml"
False
>>> extension is extension
True
```